File: desktop/src/com/dvail/clodiku/desktop/DesktopLauncher.java
package com.dvail.clodiku.desktop;

import com.badlogic.gdx.backends.lwjgl.LwjglApplication;
import com.dvail.clodiku.GameCore;

public class DesktopLauncher {
	public static void main (String[] arg) {
		new LwjglApplication(new GameCore(), "Clodiku", 400, 250);
	}
}

File: desktop/src/com/dvail/clodiku/desktop/DesktopLauncher.kt
package com.dvail.clodiku.desktop

import com.badlogic.gdx.backends.lwjgl.LwjglApplication
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration
import com.dvail.clodiku.GameCore

object DesktopLauncher {
    @JvmStatic fun main(arg: Array<String>) {
        val cfg = LwjglApplicationConfiguration()

        cfg.title = "Clodiku"
        cfg.width = 600
        cfg.height = 400

        LwjglApplication(GameCore(), cfg)
    }
}
